docs/source/what-is-mlflow.rst

```diff
@@ -0,0 +1,51 @@
+.. _what-is-mlflow:
+
+What is MLflow?
+===============
+
+MLflow is a versatile, expandable, open-source platform for managing machine learning workflows and artifacts across the machine learning lifecycle. It has predefined flavors for many popular ML libraries, but can be used with any library, algorithm, or deployment tool. It is designed to be extensible, so you can write plugins to support new workflows, libraries, and tools.
+
+.. figure:: _static/images/what-is-mlflow/mlflow-overview.png
+    :width: 100%
+
+MLflow is composed of five components:
+
+- `MLflow Tracking <https://www.mlflow.org/docs/latest/tracking.html>`_: An API and UI for logging parameters, code versions, metrics, and output files when running your machine learning code and for later visualizing the results.
```

Comment:
Perhaps changing the abstract term "output files" to the explicit components that are supported might be more helpful. "model artifacts and model environment dependencies" or something along those lines as a replacement. Might want to call out the ability to log arbitrary metadata as well.

Reply:
+1 to @BenWilson2 's comment here

the "and for later visualizing the results" also reads slightly awkwardly, IMO.